### ISSUE TITLE
No text alternative for Waveform component

### ISSUE THREAD

**Issue by adamjarling** [avalonmediasystem/structured-metadata-editor-react#66]
### Description
SME audit report line item 1.1.1, referenced in this issue:  #64 

WCAG 2.1 Checkpoint for Text Alternatives - Non-text Content.  Our structural metadata editor doesn't provide a text alternative for the waveform component.

What should this text alternative be?   Just a generic description?  There's no way to represent the content of the waveform, and it's interactional nature, through text, so is it enough to just provide some type of `aria read only` element on the component?   

### Done Looks Like
- This audit test passes in whichever tool described the failure.

### To Do
- [x] Determine what is a valid acceptance criteria for this situation?   Is anything possible to get this to pass?
- [x] Implement a solution
- [x] Re-test.

**Comment by Dananji** [comment]
Found the following options from research;
1. Use a `aria-label` for the Waveform describing the visuals
2. Use a map to make the segments accessible separately (Not sure how we can derive the coordinates for each segment dynamically)
3. Use a link to another page, giving an extensive description of the Waveform

Since we can't make the Waveform accessible through assistive technologies, I think it's best we implement a solution to describe the waveform (option 1/3) along with a `aria-readonly` attribute, and make sure that solution is accessible to a screen reader application.

@adamjarling What do you think?

**Comment by adamjarling** [comment]
@Dananji I think this seems like a good solution.

**Comment by Dananji** [comment]
Great :+1: I'm going with `aria-label` and `aria-readonly`
